index.html

```diff
@@ -29,22 +29,26 @@
                 </a>
             </div>
             <nav class="desktopnav">
-                <a href="" class="portfolio">Portfolio</a>
-                <a href="" class="portfolio">About</a>
-                <a href="" class="portfolio">Contact</a>
+                <a href="#Portfolio-menu" class="portfolio">Portfolio</a>
+                <a href="#about-menu" class="portfolio">About</a>
+                <a href="#contact-menu" class="portfolio">Contact</a>
+                <a href="mailto:fluna1973@gmail.com">
                 <i class="fa fa-envelope" aria-hidden="true" id="portfolio"></i>
+                </a>
             </nav>
-            <nav class="hamburguer">
```

Comment:
While did a good job implementing the project requirements I think it would be nice to stick tothe template design on figma. I notice the font-size in your mobile menu is too small and difficult to see please use the appropriate font-size and pay attention to detail.

This is your design:
![Screenshot from 2022-07-19 09-48-34](https://user-images.githubusercontent.com/79506105/179677200-164187a1-d4b7-4ef6-9c17-fc83186c46f8.png)

Please stick to this FIGMA design:

![Screenshot from 2022-07-19 09-48-16](https://user-images.githubusercontent.com/79506105/179677338-dc4607b4-6435-441e-9ee5-8c70d20a1a79.png)